app/controllers/v0/institution_programs_controller.rb

```diff
@@ -9,9 +9,7 @@ def autocomplete
       @data = []
       if params[:term].present?
         @search_term = params[:term]&.strip&.downcase
-        @data = InstitutionProgram.joins('INNER JOIN versions v ON v.id = institutions.version_id')
-                                  .where('v.number = ?', @version.number)
-                                  .autocomplete(@search_term)
+        @data = InstitutionProgram.autocomplete(@search_term, @version)
```

Comment:
Some of the ActiveQuery chains were in `autocomplete` and some were here.  I moved them all to `autocomplete`